modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -17,89 +15,42 @@ To configure log forwarding to an external Kafka instance, you must create a `Cl
-     outputRefs:
-     - audit-logs
-     labels:
-       logType: "audit"
-----
-<1> In legacy implementations, the CR name must be `instance`. In multi log forwarder implementations, you can use any name.
-<2> In legacy implementations, the CR namespace must be `openshift-logging`. In multi log forwarder implementations, you can use any namespace.
-<3> The name of your service account. The service account is only required in multi log forwarder implementations if the log forwarder is not deployed in the `openshift-logging` namespace.
-<4> Specify a name for the output.
-<5> Specify the `kafka` type.
-<6> Specify the URL and port of the Kafka broker as a valid absolute URL, optionally with a specific topic. You can use the `tcp` (insecure) or `tls` (secure TCP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
-<7> If you are using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must contain a `ca-bundle.crt` key that points to the certificate it represents. In legacy implementations, the secret must exist in the `openshift-logging` project.
-<8> Optional: To send an insecure output, use a `tcp` prefix in front of the URL. Also omit the `secret` key and its `name` from this output.
-<9> Optional: Specify a name for the pipeline.
-<10> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
-<11> Specify the name of the output to use when forwarding logs with this pipeline.
-<12> Optional: String. One or more labels to add to the logs.
-<13> Optional: Configure multiple outputs to forward logs to other external log aggregators of any supported type:
-** A name to describe the pipeline.
-** The `inputRefs` is the log type to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
-** The `outputRefs` is the name of the output to use.
-** Optional: String. One or more labels to add to the logs.
-
-. Optional: To forward a single output to multiple Kafka brokers, specify an array of Kafka brokers as shown in the following example:
-+
-[source,yaml]
-----
-# ...
-spec:
-  outputs:
-  - name: app-logs
-    type: kafka
-    secret:
-      name: kafka-secret-dev
-    kafka:  <1>
-      brokers: <2>
-        - tls://kafka-broker1.example.com:9093/
-        - tls://kafka-broker2.example.com:9093/
-      topic: app-topic <3>
-# ...
+    - name: app-topic 
+      inputRefs: 
+        - application
+      outputRefs:
+        - audit-logs
 ----
-<1> Specify a `kafka` key that has a `brokers` and `topic` key.
-<2> Use the `brokers` key to specify an array of one or more brokers.
-<3> Use the `topic` key to specify the target topic that receives the logs.
+<1> The name of your service account. The service account is only required in multi log forwarder implementations if the log forwarder is not deployed in the `openshift-logging` namespace.
```

Comment:
This is incorrect, it is always required:

<1> The name of your service account.